pkg/manager/rollbackconfig.go

```diff
@@ -19,7 +19,9 @@ import (
 	"github.com/onosproject/onos-config/pkg/events"
 	"github.com/onosproject/onos-config/pkg/store"
 	"github.com/onosproject/onos-config/pkg/store/change"
-	types "github.com/onosproject/onos-config/pkg/types/change/device"
+	changeTypes "github.com/onosproject/onos-config/pkg/types/change"
+	devicechangetypes "github.com/onosproject/onos-config/pkg/types/change/device"
+	"github.com/onosproject/onos-config/pkg/types/change/network"
```

Comment:
give alias `networkchangetypes` to be consistent

Reply:
done